core/src/test/java/com/google/instrumentation/stats/TagTest.java

```diff
@@ -52,9 +52,9 @@ public void testHashCode() {
 
   @Test
   public void testToString() {
-    assertThat(Tag.create(K1, V1).toString()).isEqualTo("Tag<k1,v1>");
-    assertThat(Tag.create(K2, V1).toString()).isEqualTo("Tag<k2,v1>");
-    assertThat(Tag.create(K1, V2).toString()).isEqualTo("Tag<k1,v2>");
+    assertThat(Tag.create(K1, V1).toString()).isEqualTo("Tag{key=k1, value=v1}");
+    assertThat(Tag.create(K2, V1).toString()).isEqualTo("Tag{key=k2, value=v1}");
+    assertThat(Tag.create(K1, V2).toString()).isEqualTo("Tag{key=k1, value=v2}");
```

Comment:
Should we remove the toString() tests, now that we are using AutoValue?  Another option is to just test that the output of toString() is not the default, as suggested here: http://stackoverflow.com/a/12528424